### ISSUE TITLE
vcluster version 0.15.6 is unable to build mainifest on create

### ISSUE THREAD

**Issue by ddl-retornam** [loft-sh/vcluster#1179]
### What happened?

```
 vcluster create --kubernetes-version 1.24 testvc -n testvc-ns -f vcluster.yml --expose --connect=false
info   failed to find IPv6 service CIDR: couldn't find host cluster Service CIDR ("Service "" is invalid: spec.clusterIPs[0]: Invalid value: []string{"2001:DB8::1"}: IPv6 is not configured on this cluster")
info   Chart not embedded: "open charts/vcluster-0.15.6.tgz: file does not exist", pulling from helm repository.
info   Create vcluster testvc...
info   execute command: helm upgrade testvc https://charts.loft.sh/charts/vcluster-0.15.6.tgz --kubeconfig /var/folders/zg/6tcflqkx2mz0xz0t3cc9v2sc0000gp/T/3212091812 --namespace testvc-ns --install --repository-config='' --values /var/folders/zg/6tcflqkx2mz0xz0t3cc9v2sc0000gp/T/1951085224 --values vcluster.yml
fatal  error executing helm upgrade testvc https://charts.loft.sh/charts/vcluster-0.15.6.tgz --kubeconfig /var/folders/zg/6tcflqkx2mz0xz0t3cc9v2sc0000gp/T/3212091812 --namespace testvc-ns --install --repository-config='' --values /var/folders/zg/6tcflqkx2mz0xz0t3cc9v2sc0000gp/T/1951085224 --values vcluster.yml: Release "testvc" does not exist. Installing it now.
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(StatefulSet.spec.template.spec.securityContext): unknown field "allowPrivilegeEscalation" in io.k8s.api.core.v1.PodSecurityContext
```

Which seems to be an error introduced by this change [pr-1156](https://github.com/loft-sh/vcluster/pull/1156/files#diff-7708a73d48c7e6501ac253f7ca0b5a1a64cc23c04e9236e4e4513d3f2bcf6ab9L101)

I  manually downloaded the `vcluster-0.15.5.tgz` chart and ran the following command, which worked.

```
vcluster create --kubernetes-version 1.24 testvc5 -n testvc-ns -f ../vcluster.yml --local-chart-dir . --expose
info   Creating namespace testvc-ns
info   failed to find IPv6 service CIDR: couldn't find host cluster Service CIDR ("Service "" is invalid: spec.clusterIPs[0]: Invalid value: []string{"2001:DB8::1"}: IPv6 is not configured on this cluster")
info   Create vcluster testvc5...
info   execute command: helm upgrade testvc5 . --kubeconfig /var/folders/zg/6tcflqkx2mz0xz0t3cc9v2sc0000gp/T/3500090680 --namespace testvc-ns --install --repository-config='' --values /var/folders/zg/6tcflqkx2mz0xz0t3cc9v2sc0000gp/T/1490984044 --values ../vcluster.yml
done √ Successfully created virtual cluster testvc5 in namespace testvc-ns
info   Waiting for vcluster to come up...
info   Using vcluster testvc5 load balancer endpoint: [redacted].amazonaws.com
done √ Switched active kube context to vcluster_testvc5_testvc-ns
- Use `vcluster disconnect` to return to your previous kube context
- Use `kubectl get namespaces` to access the vcluster
```

### What did you expect to happen?

I expected a new vcluster after running the command
` vcluster create --kubernetes-version 1.24 testvc -n testvc-ns -f vcluster.yml --expose --connect=false`

### How can we reproduce it (as minimally and precisely as possible)?

run the command

` vcluster create --kubernetes-version 1.24 testvc -n testvc-ns -f vcluster.yml --expose --connect=false`

### Host cluster Kubernetes version

<details>

```console
$ kubectl version
Client Version: v1.27.2
Kustomize Version: v5.0.1
Server Version: v1.24.16-eks-2d98532
```

</details>

### Host cluster Kubernetes distribution

<details>

```
# v1.24.16-eks-2d98532 EKS
```

</details>

### vlcuster version

<details>

```console
$ vcluster --version
vcluster version 0.15.6
```

</details>

### Vcluster Kubernetes distribution(k3s(default)), k8s, k0s)

<details>

```
k3s(default)
```

</details>

### OS and Arch

<details>

```
OS:  MacOS
Arch:
Darwin amd64
```
</details>

**Comment by ThomasK33** [comment]
Hey @ddl-retornam 👋 

Thanks for opening this issue, as there are two issues described here.

1. The security context getting incorrectly applied to the podSecurityContext should be fixed with #1181.
2. About the missing embedded helm charts, I will look into it and close them in another PR.

Once both are done, we'll run a patch release.

**Comment by ThomasK33** [comment]
Please check whether you can reproduce the error using the latest [0.15.7 release](https://github.com/loft-sh/vcluster/releases/tag/v0.15.7)?
